games/minimal/mods/testtools/init.lua

```diff
@@ -0,0 +1,749 @@
+local S = minetest.get_translator("testtools")
+local F = minetest.formspec_escape
+
+-- TODO: Add a Node Metadata tool
+
+local function check_priv(player, priv)
```

Comment:
I don't get the point of this at all. It's also potentially dangerous, if someone tries to check a non-whitelisted priv. You can use:

```lua
minetest.check_player_privs(player, "server")
```

to check a single privilege